scripts/binary/trigger-publish-binary-pipeline.js

```diff
@@ -14,6 +14,7 @@ const { getNextVersionForBinary } = require('../get-next-version')
   }
 
   const body = JSON.stringify({
+    branch: 'lerna-optimize-tasks',
```

Comment:
This ensures that the pipeline that we trigger runs on the `lerna-optimize-tasks` branch in the publish binary workflow. I'll remove this before merging and merge my corresponding PR https://github.com/cypress-io/cypress-publish-binary/pull/28